### PR TITLE
Rebuild NGINX image to support FIPS TLS termination

### PR DESCRIPTION
**Pull request by alavrovinfb**
NGINX image rebuilt with OpenSSL FIPS mode enabled

### CODE REVIEW

File: images/nginx/Makefile [hunk 1]
```diff
@@ -13,24 +13,26 @@
...
+SHELL = /bin/bash -o pipefail -o errexit
 
-DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DIR := $(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
 INIT_BUILDX=$(DIR)/../../hack/init-buildx.sh
 
 # 0.0.0 shouldn't clobber any released builds
-TAG ?= 0.0
-REGISTRY ?= gcr.io/k8s-staging-ingress-nginx
+TAG ?= $(DATE)-$(GIT_COMMIT)
```
Comment (burov) [inline images/nginx/Makefile:25]: Is there an ability to use tags there ? With this approach we probably have to tag release image manually, am i right ?

**Review by burov — APPROVED** [review]
Looks good, just a few confirmation regarding tags for image

File: images/nginx/Makefile (hunk 1 above)
Comment (alavrovinfb) [reply images/nginx/Makefile:25]: Not really, if `TAG` environment variable isn't set, the tag will be generated according to `TAG ?= $(DATE)-$(GIT_COMMIT)` scheme. e.g. https://hub.docker.com/layers/infoblox/nginx-fips/20200908-bcd33a8d2/images/sha256-f16dee0ab2fdfabf0f70163f071682d350970906d52fc5b730c0a413f3cdc925?context=repo

Comment (burov) [reply images/nginx/Makefile:25]: Cool, thanks for clarification

Comment (burov) [reply images/nginx/Makefile:25]: No concerns from my side

**Review by daniel-garcia — APPROVED** [review]
LGTM

**Comment by daniel-garcia** [comment]
Will you add a circleci or jenkins file to drive the build?

**Comment by alavrovinfb** [comment]
> Will you add a circleci or jenkins file to drive the build?

It hasn't been determined to create a CI pipeline, but I probably do this.

**Pull request merged**